source/spp/mnm/spp_117_nl_abiotiek_waterlopen/07_referenties.Rmd

```diff
@@ -0,0 +1,53 @@
+# Referenties {.unnumbered}
+
+Bijkerk R.
+(red) (2014) Handboek Hydrobiologie.
+Biologisch onderzoek voor de ecologische beoordeling van Nederlandse zoete en brakke oppervlaktewateren.
+Deels aangepaste versie (Rapport 2014 - 02).
+Stichting Toegepast Onderzoek Waterbeheer, Amersfoort.
+DIW (2003).
+Decreet Integraal Waterbeleid van 18 juli 2003 (B.S. 5/12/2003).
+<https://navigator.emis.vito.be/mijn-navigator?woId=75697> Hartong & Termes (2009).
```

Comment:
Hartong & Termes op een nieuwe lijn zetten

Reply:
In orde